pkg/backend/httpstate/policypack.go

```diff
@@ -156,37 +167,41 @@ func (pack *cloudPolicyPack) Publish(
 	// Publish.
 	//
 
-	fmt.Println("Uploading policy pack to Pulumi service")
+	fmt.Println("Uploading Policy Pack to Pulumi service")
 
 	version, err := pack.cl.PublishPolicyPack(ctx, pack.ref.orgName, analyzerInfo, bytes.NewReader(packTarball))
 	if err != nil {
 		return result.FromError(err)
 	}
 
-	fmt.Printf("\nPermalink: %s/policypacks/%s/%d\n", pack.Backend().URL(), pack.ref.Name(), version)
+	// Handle older versions of pulumi/policy that do not provide the version tag.
+	if analyzerInfo.Version == "" {
+		analyzerInfo.Version = strconv.Itoa(version)
+	}
 
+	fmt.Printf("\nPermalink: %s/policypacks/%s/%s\n", pack.Backend().URL(), pack.ref.Name(), analyzerInfo.Version)
```

Comment:
Should we change `PublishPolicyPack` to return a string rather than int? Then the if check here could be removed.